templates/archives/our_team.jinja2

```diff
@@ -3,227 +3,224 @@
 {% block title %}
 About the Lab
 {% endblock %}
-{% block header %}
-    <div align="center" class="typing" style="font-size:50px">
-        Our Team
-    </div>
-     &nbsp
-     &nbsp
-{% endblock %}
-
 
 {% block content %}
 
-    <div class="page-subsection">
-    <p>The Archival History of Computing project began in November, 2018 with an undergraduate cohort of students under the mentorship
-        of the staff of the Digital Humanities program at MIT. Every member
-        of the team has contributed to the project's design, development, implementation, and maintenance.</p>
-    <p>Undergraduates participated by visiting the archives, digitizing documents, creating metadata, building the website, and conducting analyses. These students
-        created all the data visualizations and stories present on this website. Every individual contribution, along with work done in small teams,
-        was integral to this digitization and contextualization of the MIT Computation Center's Archives.</p>
-    <p>You can learn more about <a href="https://digitalhumanities.mit.edu/">the Digital Humanities program at MIT</a> and check out our last project <a href="http://gendernovels.digitalhumanitiesmit.org/">Gender/Novels</a> </p>
-
+    <div align="center">
+    <h1>About Us</h1>
     </div>
     <br>
-    
-    <br>
 
+    <p>The Archival History of Computing project began in November, 2018 with an undergraduate cohort of
```

Comment:
I think I would reframe some of this to showcase the project and the entire team, and how it intersects with SHASS, Mellon, the libraries, etc. I can work on this though